ui/pages/confirmations/components/confirm/info/typed-sign/permit-simulation/permit-simulation.tsx

```diff
@@ -56,7 +56,7 @@ const PermitSimulation: React.FC = () => {
         <ConfirmInfoRowText text={t('permitSimulationDetailInfo')} />
       </ConfirmInfoRow>
       <ConfirmInfoRow label={t('approve')}>
-        <Box>
+        <Box style={{ alignSelf: 'flex-end', marginLeft: 'auto' }}>
```

Comment:
`alignSelf` probably has no affect here since children wrapped around `JustifyContent.spaceBetween` in `ConfirmInfoRow`

Reply:
just wanted to confirm so you dont have to 

https://github.com/MetaMask/metamask-extension/assets/7644512/e6e1f0ee-19b0-4fc1-b4be-6a86956aefe0

Reply:
It works here to right align the value:

<img width="317" alt="Screenshot 2024-06-20 at 6 13 09 PM" src="https://github.com/MetaMask/metamask-extension/assets/2182307/b66204f2-49d3-4b22-b9fb-c860f19c60fb">

Reply:
good catch, I updated the PR